scripts/build_openupgrade_docs

```diff
@@ -6,7 +6,8 @@ DOC_PARTS="7.0/openerp/openupgrade/doc/source/modules50-60.rst
 7.0/openerp/openupgrade/doc/source/modules60-61.rst
 7.0/openerp/openupgrade/doc/source/modules61-70.rst
 8.0/openerp/openupgrade/doc/source/modules70-80.rst
-9.0/openerp/openupgrade/doc/source/modules80-90.rst"
+9.0/openerp/openupgrade/doc/source/modules80-90.rst
+10.0/odoo/openupgrade/doc/source/modules90-100.rst"
```

Comment:
Can we put the quote in other line for avoiding 2 lines diff on each version?

Reply:
Think optimistic ;-) The optimistic way is to merge and then create a new PR yourself

Reply:
Well, I don't know very well this part. That's why I'm asking if it can be done.

Reply:
added